integration-cli/docker_cli_run_test.go

```diff
@@ -146,34 +146,6 @@ func (s *DockerSuite) TestRunWorkingDirectory(c *check.C) {
 	}
 }
 
-// pinging Google's DNS resolver should fail when we disable the networking
-func (s *DockerSuite) TestRunWithoutNetworking(c *check.C) {
```

Comment:
I should remove from here https://github.com/docker/docker/pull/17724/files#diff-0189e098e6ba3aeffd9ee321ee6aca8aL168